upstairs/src/lib.rs

```diff
@@ -2366,7 +2399,9 @@ impl Downstairs {
                 dependencies: _,
                 writes: _,
             } => {
-                panic!("WriteUnwritten not supported");
+                cdt::gw__write__unwritten__done!(|| (gw_id));
+                // XXX Should the guest see as writes?
```

Comment:
I don't think so - there shouldn't be a way for guests to trigger any `WriteUnwritten`, so it shouldn't show up as part of their traffic stats.

Reply:
Yeah..
I think the IO itself should be counted somewhere, and dtrace probes are fine for that.

Reply:
change made.